tests/handlebars-mustache-statement/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -219,18 +216,11 @@ singleQuote: true
 </div>
 
 <div
-  class="a-first-class
-  {{if this.optionOne 'optionOne'}}
-
-   {{if this.optionTwo 'optionTwo'}}
-
-   {{if this.optionThree 'optionThree'}}
-
-   {{if this.optionFour 'optionFour'}}
-
-   {{if this.optionFive 'optionFive'}}
-
-   {{this.class}}"
+  class="a-first-class {{if this.optionOne 'optionOne'}} {{
+    if this.optionTwo 'optionTwo'
```

Comment:
Is it really okay to replace new lines with spaces in attributes? I know little about Glimmer, but in plain HTML, e.g. in the `title` attribute, new lines and spaces are rendered differently.
```html
<button title="foo bar">1</button>
<button title="foo
bar">2</button>
```

Reply:
This is very true. 

And it seems that a previous PR has fixed the initial issue. Even though I've not been able to quickly bisect which one. I'm closing this PR then!